vhpc_toolkit/get_args.py

```diff
@@ -739,6 +739,53 @@ def affinity_array(string):
         type=str,
         help="Name of physical function which backs up SR-IOV Passthrough",
     )
+
+    sriov_host_parser = subparsers.add_parser(
+        "sriov_host",
+        help="Modify SR-IOV configuration on host(s).\n"
+        "This operation assumes that SR-IOV drivers have been installed on ESXi host",
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+    sriov_host_group1 = sriov_host_parser.add_mutually_exclusive_group(required=True)
+    sriov_host_group2 = sriov_host_parser.add_mutually_exclusive_group(required=True)
+    sriov_host_group1.add_argument(
+        "--host",
+        action="store",
+        default=None,
+        type=str,
+        help="Name of the host on which to perform the modify SR-IOV operation",
+    )
+    sriov_host_group1.add_argument(
+        "--file",
+        action="store",
+        default=None,
+        type=str,
+        help="Name of the file containing a list of hosts, "
+        "one per line, to perform the modify SR-IOV operation",
+    )
+    sriov_host_parser.add_argument(
+        "--device",
+        action="store",
+        default=None,
+        type=str,
+        help="Address of the SR-IOV device in format xxxx:xx:xx.x",
```

Comment:
help="PCIe address of the Virtual Function (VF) of the SR-IOV device in format xxxx:xx:xx.x"